alacritty/src/input.rs

```diff
@@ -33,13 +34,23 @@ use alacritty_terminal::vi_mode::ViMotion;
 
 use crate::clipboard::Clipboard;
 use crate::config::{Action, Binding, Config, Key, ViAction};
-use crate::event::{ClickState, Mouse};
+use crate::event::{ClickState, Event, Mouse};
+use crate::scheduler::{Scheduler, SELECTION_SCROLLING_EVENT};
 use crate::url::{Url, Urls};
 use crate::window::Window;
 
 /// Font size change interval.
 pub const FONT_SIZE_STEP: f32 = 0.5;
 
+/// Interval for mouse scrolling during selection outside of the boundaries.
+const SELECTION_SCROLLING_INTERVAL: Duration = Duration::from_millis(25);
+
+/// Minimum number of pixels at the bottom/top where selection scrolling is performed.
+const MIN_SELECTION_SCROLLING_HEIGHT: i32 = 5;
+
+/// Number of pixels for increasing the selection scrolling speed factor by one.
+const SELECTION_SCROLLING_STEP: i32 = 25;
```

Comment:
I'd lower this value tbqh, also, it could make sense to scale it with DPI as well, but I'm not sure about that.

Reply:
I think 25 is a good value to prevent it from accidentally just shooting the user way up in history. Especially because it's sometimes harder to get back since you might have more space on one side than the other.